packages/editor/src/components/post-title/index.js

```diff
@@ -149,7 +149,13 @@ function PostTitle( _, forwardedRef ) {
 			plainText,
 		} );
 
-		if ( typeof content !== 'string' && content.length ) {
+		event.preventDefault();
```

Comment:
There's a duplicate `preventDefault` call inside the `if ( typeof content !== 'string' )` block that can be removed.